### ISSUE TITLE
PeepOpen now triggers a dialog "foo.rb is a script application"

### ISSUE THREAD

**Issue by mmower** [topfunky/PeepOpen-Issues#260]
This started sometime last week I think and may be related to having installed PeepOpen 0.3.5 (I'm sure I updated around the time I started seeing this, may be coincidence though).

When I use ,p from MacVim 7.3(57) on OSX 10.6.8, PeepOpen 0.3.5 and open a Ruby file I get this dialog prompt appear:

  http://cl.ly/3n3b1k0Z2Q023M0Y3o1e

Is it maybe something I have done?

Any help welcome.

Thanks.

Matt

**Comment by topfunky** [comment]
I believe this happens when OS X thinks you downloaded an executable file from the Internet. I've experienced it sometimes, too.

Do you see it repeatedly for the same file, or just once per file?

One solution is to mark the files as non-executable in the Terminal.

**Comment by mmower** [comment]
Yes it's the same thing that happens when you've downloaded and try to open something but, in this case, it's source files that are local, not downloaded. However it doesn't happen every time so there's something odd going on. I guess it's possibly related to files cloned by Git or copied by Dropbox.

Next time it happens I will take a close look at the file and where it came from.

Matt
